test_conformance/buffers/test_buffer_fill.cpp

```diff
@@ -586,8 +585,6 @@ int test_buffer_fill( cl_device_id deviceID, cl_context context, cl_command_queu
     loops = (loops < 5 ? loops : 5);
     for (i = 0; i < loops; i++)
     {
-        ii = i << 1;
```

Comment:
Think the `int ii` definition on line 571 can be deleted now too, LGTM otherwise